mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelStore.kt

```diff
@@ -105,8 +105,8 @@ class MvRxViewModelStore(private val viewModelStore: ViewModelStore) {
             val holder = viewModelsState.getParcelable<MvRxPersistedViewModelHolder>(key)
                 ?: throw IllegalStateException("ViewModel key: $key expected to be in stored ViewModels but was not found.")
             map[key] = when (host) {
-                is Fragment -> restoreViewModel(host, holder, arguments)
-                is FragmentActivity -> restoreViewModel(host, holder, arguments)
+                is Fragment -> restoreViewModel(key, host, holder, arguments)
```

Comment:
I think it would be great to delete this class if we can now (and move these helper functions into a file just for dealing with saved state.

Reply:
I'm able to actually delete this class and `MvRxViewModelStoreOwner` as well!